File: Code/Mobile/Android/EvolveChain_Android/app/src/main/java/com/newgen/evolvechain/models/documnets/UtilityBillTypeModel.java
package com.newgen.evolvechain.models.documnets;

/**
 * Created by onkar.gupta on 5/31/2018.
 *
 */

public class UtilityBillTypeModel {
    private String code, name;

    public UtilityBillTypeModel(String code, String name){
        this.code = code;
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public String getCode() {
        return code;
    }
}

File: Code/Mobile/iOS/EvolveChain/Models/SignupConfig/UtilityBillTypeModel.swift
//
//  UtilityBillModel.swift
//  EvolveChain
//
//  Created by Abhay Shankar on 31/05/18.
//  Copyright © 2018 EvolveChain. All rights reserved.
//

import UIKit

class UtilityBillTypeModel: NSObject {

    var key: String             = ""
    var name: String                   = ""
    
    // MARK:- Init
    override init() {
        super.init()
        key              = ""
        name                    = ""
    }
    

    
    func initWithDictionary(billDict:Dictionary<String, Any>){
        
        self.key = RawdataConverter.string(billDict["CODE"])
        self.name = RawdataConverter.string(billDict["NAME"])
    }
    
   
}
